compiler/test/dotty/tools/dotc/CompilationTests.scala

```diff
@@ -204,8 +204,9 @@ class CompilationTests extends ParallelTesting {
       defaultOutputDir + libGroup + "/src/:" +
       // as well as bootstrapped compiler:
       defaultOutputDir + dotty1Group + "/dotty/:" +
-      // and the other compiler dependecies:
-      Jars.dottyInterfaces + ":" + Jars.jlineTerminal + ":" + Jars.jlineReader,
+      // and the other compiler dependenies:
+      Properties.compilerInterface + ":" + Properties.scalaLibrary + ":" + Properties.scalaAsm + ":" +
+      Properties.dottyInterfaces + ":" + Properties.jlineTerminal + ":" + Properties.jlineReader,
```

Comment:
Let's use `TestConfiguration.mkClasspath`?

Reply:
mkClasspath checks that each directory exists so won't work on libGroup and dotty1Group here. I agree this code would be cleaned up more but let's keep that for another pr.

Reply:
Ok. Then we should use `File.pathSeparator` here as well

Reply:
There's a lot of place where we use ":" right now which need to be fixed I've opened an issue for it: https://github.com/lampepfl/dotty/issues/5000